test/tests/imageserver/test_os_image_service.py

```diff
@@ -0,0 +1,411 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+This test validate the image-service of RackHD. It use OS iso file stored in remote server server to load on
+the image-service server. This test will create os repo in the server and then go through the repo directories and
+compare files&directories with the local iso image. This test focus on image service OS repo functionality itself.
+You need put an config file in the /config directory
+'''
+
+import os
+import sys
+import urllib2
+import flogging
+import requests
+import json
+import fit_common
+import pexpect
+import unittest
+import test_api_utils
+from nose.plugins.attrib import attr
+logs = flogging.get_loggers()
+
+
+try:
+    FILE_CONFIG = json.loads(open(fit_common.CONFIG_PATH + "fileserver_config.json").read())
+except BaseException:
+    logs.error(
+        "**** Global Config file: " +
+        fit_common.CONFIG_PATH +
+        "FILE_CONFIG.json" +
+        " missing or corrupted! Exiting....")
+    sys.exit(255)
+
+
+@attr(all=True, regression=False, smoke=False)

```

Comment:
add a group here for imageservice=True (or something good).
That way we can set up a jenkins job to target this suite specifically.